Configuration/App/DuckDuckGoAppStore.xcconfig

```diff
@@ -22,6 +22,7 @@ PRODUCT_BUNDLE_IDENTIFIER = $(MAIN_BUNDLE_IDENTIFIER)
 CODE_SIGN_ENTITLEMENTS = DuckDuckGo/DuckDuckGoAppStore.entitlements
 CODE_SIGN_ENTITLEMENTS[config=Debug][arch=*][sdk=*] = DuckDuckGo/DuckDuckGoAppStoreDebug.entitlements
 CODE_SIGN_ENTITLEMENTS[config=CI][arch=*][sdk=*] = DuckDuckGo/DuckDuckGoAppStoreCI.entitlements
+CODE_SIGN_ENTITLEMENTS[config=Review][arch=*][sdk=*] = DuckDuckGo/DuckDuckGoAppStoreCI.entitlements
```

Comment:
I'd appreciate if we do either of the two:
1. Add a comment here explaining why AppStoreCI is used for App Store Review builds
2. Add a dedicated entitlements file called AppStoreReview
Also I think we're going to request the entitlement for the review AppStore app later so this is temporary? Then a comment should work.